leaflet/index.d.ts

```diff
@@ -153,16 +157,15 @@ declare namespace L {
         getSouth(): number;
         getEast(): number;
         getNorth(): number;
-        contains(otherBounds: LatLngBoundsExpression): boolean;
-        contains(latlng: LatLngExpression): boolean;
+        contains(otherBoundsOrLatLng: LatLngBoundsExpression | LatLngExpression): boolean;
         intersects(otherBounds: LatLngBoundsExpression): boolean;
         overlaps(otherBounds: BoundsExpression): boolean; // investigate if this is really bounds and not latlngbounds
         toBBoxString(): string;
         equals(otherBounds: LatLngBoundsExpression): boolean;
         isValid(): boolean;
     }
 
-    export type LatLngBoundsLiteral = Array<LatLngTuple>;
+    export type LatLngBoundsLiteral = LatLngTuple[];
 
     type LatLngBoundsExpression = LatLngBounds | LatLngBoundsLiteral;
```

Comment:
There are two overloads below that TSLint is having a hissy about - you can either combine them into one overload, or also suppress the rule for `unified-signatures` as well. Since you know the library better, I'll defer to your judgement.

(You can also keep the suppression local to these overloads with a TSLint comment.)